qt/scientific_interfaces/Inelastic/Analysis/IndirectFitDataView.cpp

```diff
@@ -23,6 +25,8 @@ const QString REAL_RANGE = "(" + REAL_NUMBER + COMMA + REAL_NUMBER + ")";
 const QString MASK_LIST = "(" + REAL_RANGE + "(" + COMMA + REAL_RANGE + ")*" + ")|" + EMPTY;
 } // namespace Regexes
 
+QString makeNumber(double d) { return QString::number(d, 'f', 6); }
```

Comment:
Can you create a constant variable at the top of the file for `6` as it is used in two places that I can see in this file?